test/test_io.py

```diff
@@ -131,3 +131,12 @@ def test_read_color_off():
 
     assert_points_xyz(color_off)
     assert_points_color(color_off)
+
+def test_read_stl_ascii():
+    stl = PyntCloud.from_file(data_path + '_ascii.stl')
+    assert_points_xyz(stl)
```

Comment:
One last problem here is that the order of the points and mesh is all messed because of how we had to extract the information from `numpy-stl`.

Just call the function from_file and don't perform any assertion for now.